core/plugins/org.polarsys.capella.core.data.requirement.properties/.project

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can you remove this. 
i think it's because you still have it open in your workspace even if it's now empty

Reply:
Didn't see it got staged, removed it